regression/strings-smoke-tests/java_string_printable_knownbug/test_string_printable.java

```diff
@@ -0,0 +1,15 @@
+public class test_string_printable
+{
+   public static void main(String x)
+   {
+      String s = new String("Abc");
+      String u = "bc";
+      String t = "ab";
+      assert(s.contains(u));
+      assert(!s.contains(t));
+
+      String z = new String(x);
+      if (z.length() > 3) assert(t.contains(z));
```

Comment:
Even though it is a test, probably better to be consistent with the Java coding standards:

```java
if (z.length() > 3)
  assert(t.contains(z));
```